### ISSUE TITLE
Navigation arrows cover content 

### ISSUE THREAD

**Issue by mivoligo** [ubuntu/codelabs#7]
The left and right navigation arrows cover text on the pages. Screenshot with example attached.
![screenshot at 15-26-45](https://cloud.githubusercontent.com/assets/1629120/20488500/2752b732-afff-11e6-9363-482f2160606c.png)

**Comment by didrocks** [comment]
This depends on your browser size, correct? and if you scroll down, then, you can copy the content and read it all, correct?

I'm unsure where we would put them if not in a fixed position?

**Comment by mivoligo** [comment]
Yes, it does depend on the browser size. It's true that in the case visible on the screenshot, I can scroll down to see the text to copy. But generally the last line of text which is at the bottom of the page is covered by the navigation arrows even if I scroll down maximally.

Possible solutions:
- make sure the page content is above the arrows when scrolled down (move the arrows down a bit)
- make the navigation arrows semi-transparent

**Comment by didrocks** [comment]
oh, do you have particular codelabs steps when that happens? I think we can add some general padding at the bottom if needed.

**Comment by mivoligo** [comment]
In "Create your first snap" steps: 4,5,6,8
In "Basic snap usage" steps: 2,3,4,5

**Comment by mivoligo** [comment]
Interestingly, in the narrow view (without the navigation on the left hand side), the padding is just fine. When I scroll down, the arrows are below the text.

**Comment by mivoligo** [comment]
![zrzut ekranu z 2016-11-30 10-09-24](https://cloud.githubusercontent.com/assets/1629120/20748256/5819dcfa-b6e5-11e6-829c-5b35e7d4985f.png)

**Comment by mivoligo** [comment]
![zrzut ekranu z 2016-11-30 10-09-40](https://cloud.githubusercontent.com/assets/1629120/20748259/5ebf422a-b6e5-11e6-96b1-6eebda2efbef.png)

**Comment by didrocks** [comment]
Yes! Ok, I'll add a little bit of padding to fix it in the coming days. Thanks for your help and feedback! :)

**Comment by mivoligo** [comment]
awesome! thanks for working on this!

**Comment by Apeshere** [comment]
![image](https://cloud.githubusercontent.com/assets/19166875/20749631/01fac4e6-b6eb-11e6-8864-c43be44c21f3.png) (screenshot page link:http://localhost:8123/codelabs/create-first-snap#5)
just to add an additional screenshot on the situation, scrolled all the way down, but still can't see one of the word in the last sentence. Same thing happens from step 3 to step 6, and step 8 in the create your first snap codelab.  

here is my browser info:
Browser window width:	1535
Browser window height:	786
Screen width:	1600
Screen height:	900
Screen color depth:	24
